demux/demux_mkv.c

```diff
@@ -1688,7 +1688,24 @@ static int demux_mkv_open_video(demuxer_t *demuxer, mkv_track_t *track)
         sh_v->rotate = rotate;
     }
 
+    // our vessel to transport global track side data to decoder context
+    sh_v->lav_codecpar = mp_codec_params_to_av(sh_v);
+    MP_HANDLE_OOM(sh_v->lav_codecpar);
+
     if (track->dovi_config) {
+        size_t dovi_size;
+        AVDOVIDecoderConfigurationRecord *dovi = av_dovi_alloc(&dovi_size);
+        MP_HANDLE_OOM(dovi);
+
+        memcpy(dovi, track->dovi_config, dovi_size);
+
+        MP_HANDLE_OOM(
```

Comment:
This shouldn't be changed from previous code. Please restore previous error check. This is not critical error. External api calls shouldn't be treated as OOM, except explicit allocation functions.